textattack/search_methods/attack.py

```diff
@@ -139,6 +139,10 @@ def _get_examples_from_dataset(self, dataset, num_examples=None, shuffle=False,
         
         if shuffle:
             random.shuffle(dataset.examples)
+
+        if num_examples <= 0:
+            return
+            yield
```

Comment:
I don't think this `yield` is supposed to be here

Reply:
I included `yield` after seeing this: https://stackoverflow.com/questions/13243766/python-empty-generator-function

I'm not familiar with generators, so I'm not sure which is correct.

Reply:
Whoa, that's weird. I think `return []` would work fine here but the generator syntax is good too! We can leave it.